libs/blog/ad-banner/ui/src/lib/instances/al-autumn-camp-banner.component.ts

```diff
@@ -0,0 +1,39 @@
+import { ChangeDetectionStrategy, Component, signal } from '@angular/core';
+
+import { AdImageBannerComponent } from '../ad-image-banner/ad-image-banner.component';
+import { AlInfiniteSliderDirective } from '../infinite-slider-directive/al-infinite-slider.directive';
+
+@Component({
+  selector: 'al-autumn-camp-banner',
+  standalone: true,
+  imports: [AdImageBannerComponent, AlInfiniteSliderDirective],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="overflow-hidden">
+      <div class="flex">
+        <al-ad-image-banner
+          *alInfiniteSlider="let banner of banners()"
+          class="flex-shrink-0 flex-grow-0 basis-full"
+          [banner]="banner"
+        ></al-ad-image-banner>
+      </div>
+    </div>
+  `,
+})
+export class AlAutumnCampBannerComponent {
+  // todo: hardcoded banners
+  readonly banners = signal([
```

Comment:
`alt` is empty. Could you append `protected` access modifier ?

Reply:
you can add them just in case, if anything won't go as planned